core/src/main/scala/zio/logging/LoggingFormat.scala

```diff
@@ -0,0 +1,6 @@
+package zio.logging
+
+import zio._
+trait LoggingFormat[Message] {
```

Comment:
Why does this exist? It is a proliferation of interfaces.

Reply:
this is the idea from ```log4zio``` that you mentioned in the issue that might be interesting. in ```log4zio``` that would be ```LogMedium``` in the future we can add those ```contramap``` to convert one to another.

Reply:
let me know I can get rid of this and back to simple interface

Reply:
This format also remove coupling to MDC because it let us control format of message.

```
override def format(message: String): ZIO[Any, Nothing, String] =
          loggerContext
            .get(correlationId)
            .map(correlationId => stringFormat.format(correlationId.getOrElse("undefined"), message))
            .provide(self)
```